src/shared/components/footer-menu/footer-menu.css.ts

```diff
@@ -0,0 +1,29 @@
+import { style } from "@vanilla-extract/css";
+
+import { themeVars } from "@shared/styles/theme.css";
+
+export const container = style({
+  paddingLeft: "0.6rem",
```

Comment:
```suggestion
  padding: "0.2rem 0 0.2rem 0.6rem",
```

<img width="582" height="177" alt="Image" src="https://github.com/user-attachments/assets/a320670e-c9b8-43ac-b115-5df878726ee9" />
상하 패딩이 존재하는것 같네요!

Reply:
<img width="745" height="514" alt="스크린샷 2025-11-21 오후 3 04 10" src="https://github.com/user-attachments/assets/049d4b0d-a17e-48ad-9376-a87b7acd3088" />

코멘트로 인한 텍스트 폰트 스타일 추가로 인하여 패딩 영역이 바뀌었고,, 확대해보니까 상하 패딩 사이에 또 여백이 존재하여,, 디자인팀과 상의 후 수정해놓겠습니다!